o8g/Extractor/config.xml

```diff
@@ -17,19 +17,22 @@
     <property octgn_name="Clan" type="property" value="clan" capitalize="True" />
     <property octgn_name="Text" type="property" value="text" isRich="True"/>
     <property octgn_name="Type" type="property" value="type" capitalize="True" />
-    <property octgn_name="Traits" type="property" value="traits" capitalize="True" delimiter=". " >
-        <replace match="yojimbo" value="Yōjimbō" />
-        <replace match="meishodo" value="Meishōdō" />
-        <replace match="kiho" value="Kihō" />
-        <replace match="daimyo" value="Daimyō" />
+    <property octgn_name="Traits">
+       <run type="property" value="traits" capitalize="True" delimiter=". ">
+          <replace match="yojimbo" value="Yōjimbō" />
+          <replace match="meishodo" value="Meishōdō" />
+          <replace match="kiho" value="Kihō" />
+          <replace match="daimyo" value="Daimyō" />
+       </run>
+       <run type="string" value="." />
```

Comment:
Think the literal value here is also causing all cards with empty traits to have a value of '.'

Any way it could be conditional on traits being present?